Monika After Story/game/script-compliments.rpy

```diff
@@ -619,3 +619,32 @@ label mas_compliment_bestgirl:
     m 5ekbfa "But if you ask me, you made the right choice."
     m 5hubfa "...and I'll be forever grateful that you did~"
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_compliments_database,
+            eventlabel="mas_compliment_lookuptoyou",
+            prompt="I look up to you!",
+            unlocked=True
+        ),
+        code="CMP",
+    )
+label mas_compliment_lookuptoyou:
+    if not renpy.seen_label("mas_compliment_lookuptoyou_2"):
+        call mas_compliment_lookuptoyou_2
+    else:
+        call mas_compliment_lookuptoyou_3
+    return
+label mas_compliment_lookuptoyou_2:
+    m 1wud "You... You look up to me?"
+    m 5hubfb "Thank you so much, [player]!"
+    m 1rkbfb "You have no idea how happy it makes me to hear you say that..."
+    m 3hubfb "I'll always look up to you, too, [player]!"
+    m 1ekbfa "I love you~"
+label mas_compliment_lookuptoyou_3:
```

Comment:
Looks like both paths you have here are basically identical. It doesn't look like you have any affection gain in here, so you don't need to have the 3 sections like some other compliments do. 

You could just have it act as a regular topic in the compliments file like bestgirl, pong, and chess.

Reply:
Oh, crap! I forgot to add affection gain!

Reply:
If you add affection, especially if it bypasses the cap, we'll have to be a bit more strict with the reviews.

Reply:
I'm okay with a strict review. I want it to be good enough, of course!